c++/src/kj/exception.c++

```diff
@@ -58,9 +58,21 @@
 #include <cxxabi.h>
 #endif
 
+// Use KJ_HAS_BACKTRACE; KJ_HAS_BACKTRACE is for backwards compat only.
+#if !defined(KJ_USE_BACKTRACE)
+#if defined(KJ_HAS_BACKTRACE) && ((KJ_HAS_BACKTRACE + 0) == 1)
+#define KJ_USE_BACKTRACE 1
+#endif
+#endif
+
+#ifndef KJ_USE_BACKTRACE
 #if (__linux__ && __GLIBC__ && !__UCLIBC__) || __APPLE__
-#define KJ_HAS_BACKTRACE 1
-#include <execinfo.h>
+#define KJ_USE_BACKTRACE 1
+#endif
+#endif
+
+#if KJ_USE_BACKTRACE
+# include <execinfo.h>
```

Comment:
Nitpick: This style doesn't appear anywhere else in the codebase, let's be consistent.

```suggestion
#include <execinfo.h>
```